### PR TITLE
Add a filter to the overview screen to show only the topics and queues with deadletters

### PR DESCRIPTION
**Pull request by crejb**
Add a filter to the overview screen to show only the topics and queues with deadletters
sbmanager filter 1
sbmanager filter 2

### CODE REVIEW

File: src/SbManager/Content/tmpl/manager/home.html [hunk 1]
```diff
@@ -8,17 +8,38 @@ <h1 id="type"><i class="sbmicon sbmicon-logo"></i> SbManager</h1>
     <loading ng-if="!model"></loading>
 
     <div ng-if="model">
+        <div class="row" ng-if="model.TotalDeadLetters > 0">
+            <div class="panel">
+                <div class="panel-body">
+                    <div>
+                        <span message-type-icon message-type="messageTypes.dead" />
+                        <span ng-if="model.TotalDeadLetters == 1">There is currently 1 deadletter.</span>
```
Comment (drewfreyling) [inline src/SbManager/Content/tmpl/manager/home.html]: `ng-pluralize` might work a bit better here?

**Comment by drewfreyling** [comment]
@AmosC @rovens @zhangyiping What do you guys think about this PR?

**Comment by lukeschafer** [comment]
My 2c: 

Great idea for a feature. Perhaps instead of using extra real estate you can incorporate it into the UI, along the lines of:

image

And for the toggle, a checkbox in the fixed footer, maybe on the right?

**Comment by drewfreyling** [comment]
Thanks @lukeschafer. I think that might be a bit of a better user experience along the lines of that. What do you think @crejb?

**Comment by jonyeezs** [comment]
I like the idea of the totals closer to the column.

For filter, I don't mind it being top. But some sort visible input should be associated, ie a checkbox.

**Comment by lukeschafer** [comment]
Yeah sure, it's certainly not bad to have it where you put it, but 2 things to think about are:

1. Introducing a new paradigm for one feature, where a similar paradigm (actions that interact with the page) already exists
2. Vertical realestate. Most places have tons of queues/topics, so vertical realestate is at a premium. A checkbox by itself could only take about 20 pixes, but to make it look good and flow well you need padding, margins, maybe a header etc. That all adds up.

I'm just a sideline observer though ;)

**Comment by crejb** [comment]
@drewfreyling good call on ng-pluralize, thanks

@lukeschafer the totals in the column header looks great.  Regarding vertical space, I definitely agree it is at a premium. In a way, the whole purpose for this feature request is to target this by reducing the need to scroll down through the massive number of queues/topics to find the few with deadletters. Personally this is the only use case I ever have for SBManager but I can see it would be an interruption for other use cases.

I created a new story to put the totals in the column headers (https://github.com/GlobalX/SbManager/issues/80) so we can focus on the filter here. I'll have another think about a more concise way to show the filter. @jonyeezs lets chat about your checkbox idea.

**Comment by jonyeezs** [comment]
> @jonyeezs lets chat about your checkbox idea.

Something along:

- [ ] Show Only Deadletters

and maybe on top of the headers

**Comment by drewfreyling** [comment]
I also think we should add a route in for this as well so we can have it displayed on a monitor somewhere.  Happy for that to be outside the scope of this PR.

**Comment by jonyeezs** [comment]
Let's just get this in.

LGTM!

**Comment by drewfreyling** [comment]
Closing because it is superseeded by #84

**Pull request closed without merging**